keystone_spassword/contrib/spassword/core.py

```diff
@@ -25,8 +25,7 @@
 from keystone import auth
 from keystone.auth.plugins import password
 from keystone.common import dependency
-from keystone.common import extension
-from keystone.common import wsgi
+#from keystone.common import wsgi
```

Comment:
Leftover comment?

Reply:
Fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/57/commits/09846cb1bead252b70340d4a8f033c750aa0cd3f